### PR TITLE
Create whom.md

### PR DESCRIPTION
**Pull request by thelifeandtimes**

### CODE REVIEW

**Comment by thelifeandtimes** [comment]
dependent on https://github.com/urbit/urbit.org/pull/1753 being merged.

**Comment by gordoncc** [comment]
Looks like there's an issue with the build.  @matildepark do you know what's up?

**Comment by thelifeandtimes** [comment]
this should align with what you just merged; might be the caching issue rearing it's ugly head?

**Comment by matildepark** [comment]
I do not make mistakes. Master just needed to be merged into this branch.

**Comment by matildepark** [comment]
Well, the current site build is failing because these are strings, so I'll make a PR for those two commits.

File: content/grants/whom.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+[taxonomies]
+grant_type = ["Proposal"]
+grant_category = ["App Dev"]
+
+[extra]
+image = ""
+description = "A dedicated contacts app to store contact info that other apps can programmatically access"
+reward = "3 stars"
+assignee = ["~rivmud-fabwen"]
+champion = ["~palfun-foslep"]
```
Comment (Fang-) [inline content/grants/whom.md]: imposter spotted!

```suggestion
champion = ["~palfun-foslup"]
```

**Review by Fang- — COMMENTED** [review]
Needs to have my name on it, but otherwise looks like the proposal that was discussed & deemed fine on the network. The description gives more weight to the programmatic access than the grant does, but maybe nbd.

**Comment by thelifeandtimes** [comment]
@gordoncc this can be merged.

**Comment by gordoncc** [comment]
Thank you urbit gods this is the first thing I wanted after going to meetups in 2019

**Review by gordoncc — APPROVED** [review]

**Pull request merged**